### ISSUE TITLE
MultiLoader version

### ISSUE THREAD

**Issue by Skidamek** [Fallen-Breath/fabric-mod-template#1]
### Motivation

This template is already pretty impressive, however it would be super cool if we could somehow do forge/fabric/(quilt) version of it on a different branch. That would be amazing to make multi version & loader mods so easily.

### Description

I think we can achieve that by using [architectury loom](https://github.com/architectury/architectury-loom) instead of fabric one.  I think that architectury also support yarn for forge that would be amazing to have same mappings across mod loaders.

**Comment by Skidamek** [comment]
I know about essential gradle which does something similar but they somewhy using their own fork outdated version of architectury loom with bugs. Also they also for some reason use different mappings between two mod loaders. This would be ultimate Minecraft mod template + implement forgix into it lol amazing...

**Comment by Fallen-Breath** [comment]
yeah I'm aware of the architectury thing. it's on the todo list, but I can't guarantee that i can do make thing work correctly tho

**Comment by kikugie** [comment]
What are the current known issues with the architectury test branch?

**Comment by Fallen-Breath** [comment]
I have no idea how exactly the `architectury-test` branch works, and idk if I can maintain that branch correctly and healthily. There might be some hidden issues introduced by mixing multiple Minecraft versions in a single project that no one knows

There's a time when the mod development refused to launch the forge client / server, displaying `java.lang.NoClassDefFoundError: jdk/nashorn/api/scripting/NashornScriptEngineFactory`, and I have no idea why. It was fixed by randomly reloading the gradle project and running the architectury example mod, but I don't know what was happening at all

Also, the `architectury-test` branch currently does not support the Platform Specific APIs or similiar features from the architectury-plugin. If your codes are very platform-dependent, then writing them with the current template will be a mess

**Comment by RubixDev** [comment]
I just stumbled across this. Anyone interested might want to take a look at how I configured [Inventorio](https://github.com/RubixDev/Inventorio) and/or at https://github.com/tr7zw/ProcessedModTemplate and https://github.com/tr7zw/3d-Skin-Layers
